docs/release.md

```diff
@@ -0,0 +1,36 @@
+---
+title: BTP Manager release pipeline
+---
+
+## Overview
+
+The BTP Manager release pipeline creates proper artifacts:
+ - btp-operator module OCI image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/btp-manager)
+ - btp-manager Docker image in the [registry](http://europe-docker.pkg.dev/kyma-project/prod/unsigned/component-descriptors/kyma.project.io/module/btp-operator)
+ - template.yaml
+
+## Run the pipeline
+
+### Create a release
+To create a release, follow these steps:
+
+1. Run GitHub action **Create a release**: 
+    i. go to the **Actions** tab
+    ii. click on **Create a release** workflow 
+    iii. click  **Run workflow** on the right
+    iv. provide a version, for example, 1.2.0.
+2. The GitHub action, defined in the `.github/workflows/create-release.yaml` file, creates a GitHub tag and release with the provided name.
+3. The tag creation triggers Prow Jobs, `post-btp-manager-module-build` and `post-btp-manager-build`, defined in [btp-manager-build.yaml](https://github.com/kyma-project/test-infra/blob/main/prow/jobs/btp-manager/btp-manager-build.yaml).
+4. `post-btp-manager-build` builds a Docker image tagged with the release name.
+5. `post-btp-manager-module-build` runs the `kyma alpha create module` command, which creates a Kyma module, and pushes the image to the registry. Finally, the job uploads the `template.yaml` file to the btp-manager release as a release asset.
+6. The GitHub action waits for the `template.yaml` asset in the GitHub release and for images in the Docker registry.
```

Comment:
in `release directory`?

Reply:
github release like this: https://github.com/kyma-project/btp-manager/releases/tag/0.0.19